runner/ollamarunner/runner.go

```diff
@@ -773,6 +779,7 @@ func (s *Server) computeBatch(activeBatch batchState) {
 		if !flushPending(seq) {
 			s.removeSequence(i, llm.DoneReasonConnectionClosed)
 		}
+		seq.samplingDuration += time.Since(seq.lastUpdatedAt)
```

Comment:
Somewhat similar to above, I think this will capture the sampling time for this sequence and the ones before it but the sampling time for later sequences will get counted in the eval time.